### ISSUE TITLE
ImportError: No module named simplenote

### ISSUE THREAD

**Issue by alechstong** [mrtazz/simplenote.vim#106]
I installed simplenote.vim using Vim Plug.
my python version is 3.7.4.
vim version: 8.1.1776

I tried `pip install simplete`, but it doesn't make any difference. I have it working on my desktop with very similar setup, but it is not working on my laptop.

Please help.

The following is the error output when I ran `:SimplenoteList`
```
Error detected while processing /home/alex/.vim/plugged/simplenote.vim/autoload/simplenote.vim:

line  200:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/alex/.vim/plugged/simplenote.vim/autoload/SimplenoteUtilities.py", line 8, in <mo
dule>
    import simplenote
ImportError: No module named simplenote
Error detected while processing function simplenote#SimplenoteList:
line    2:
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/home/alex/.vim/plugged/simplenote.vim/autoload/SimplenoteList.py", line 15, in <module
>
    except simplenote.SimplenoteLoginFailed:
NameError: name 'simplenote' is not defined

```

**Comment by atomicules** [comment]
Hi. Simplenote should get installed via a git submodule. It's possible Vim Plug doesn't support that. I shall do some investigation and get back to you.

**Comment by alechstong** [comment]
Hi atomicules,

Thank you for your prompt reply. Here is my [vimrc](https://paste.ee/p/gxiPN) in case you wanna take a look.

**Comment by alechstong** [comment]
Hi atomicules,

I managed to get it working again.

Here's what I did if anyone's interested:

1. remove all plugins from vimrc, source it and run `:PlugClean`

2. reinstall simplenote.vim, and see if it is working. (In my case, it is)

3. install the rest of the plugins  one by one, and see if any of them causes the error. (I was expecting that one of them would trigger the error, but none of them did)

I guess it is some bug of vim-plugin when installing a bunch of plugins together in batch. Because I tried reinstall all plugins, and it didn't work.
